lib/proxy/plugin_validators.rb

```diff
@@ -7,6 +7,7 @@ def initialize(plugin, setting_name)
 
     def validate!
       setting_value = @plugin.settings.send(@setting_name)
+      return true if setting_value.nil?
```

Comment:
It's a bit too clever, which I don't like: my thinking was that this is used together with default setting for the same parameter, and would require a deliberate effort to set the value to nil via config file.  I wonder if supporting a conditional here would be more clear?